### DISCUSSION TITLE
New element: Reading time

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#24732] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

A simple but important widget would be great for content driven websites.

### Describe the solution you'd like.

Reading time element with 3 options :

- Content selector
- Prefix and Suffix
- Words per minutes

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by ayoubkhan558** [comment]
Yes need, also add dynamic tag for reading time.
